### PR TITLE
Add ec2 instances

### PR DESCRIPTION
**Pull request by bmacharia**

### CODE REVIEW

File: instance.tf [hunk 1]
```diff
@@ -21,7 +21,7 @@ resource "aws_security_group" "public" {
     from_port   = 22
     to_port     = 22
     protocol    = "tcp"
-    cidr_blocks = ["98.56.189.252/32"]
+    cidr_blocks = ["76.137.140.36/32"]
```
Comment (vladimirmukhin) [inline instance.tf]: Looks like your public IP changes often. Let's have it as a variable
```suggestion
    cidr_blocks = ["${var.my_public_ip}/32"]
```

File: provider.tf [hunk 2]
```diff
@@ -9,9 +6,10 @@ terraform {
...
-# provider block supplies the info needed to access aws
+
 provider "aws" {
   region                   = "us-west-2"
   shared_credentials_files = ["~/.aws/credentials"]
   profile                  = "kenna"
-}
\ No newline at end of file
+}
+
```
Comment (vladimirmukhin) [inline provider.tf]: Looks like you have more than one newline at the end of the file.
Let's have only one.

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

File: provider.tf (hunk 2 above)
Comment (bmacharia) [reply provider.tf]: Deleted extra line at end of file

File: instance.tf (hunk 1 above)
Comment (bmacharia) [reply instance.tf]: Changed hard coded ip address to a variable

**Review by vladimirmukhin — APPROVED** [review]

**Pull request merged**